hypertrace-view-generator/src/main/resources/configs/view-gen-raw-service/application.conf

```diff
@@ -18,10 +18,12 @@ flink.source {
   input.class = org.hypertrace.core.datamodel.StructuredTrace
   schema.registry {
     schema.registry.url = "http://localhost:8081"
+    schema.registry.url = ${?SCHEMA_REGISTRY_URL}
     specific.avro.reader = true
   }
   kafka {
     bootstrap.servers = "localhost:9092"
+    bootstrap.servers = ${?KAFKA_BOOTSTRAP_SERVERS}
```

Comment:
we should be using same env for kafka. `KAFKA_BOOTSTRAP_SERVERS` or `KAFKA_BROKER_LIST`

Reply:
I had similar thoughts but was going with the variable names in the config itself.  But, this makes sense too - 1. same name 2. in case if we combine, one less env variable. Will change KAFKA_BROKER_LIST -> KAFKA_BOOTSTRAP_SERVERS

Reply:
Done.